scipy/special/_basic.py

```diff
@@ -107,6 +107,75 @@ def _nonneg_int_or_fail(n, var_name, strict=True):
     return n
 
 
+def sinc(x):
+    r"""
+    .. deprecated:: 1.12.0
+        scipy.special.sinc is deprecated in SciPy 1.12 and will be removed
+        in SciPy 1.14. As scipy.special.sinc is an alias of numpy.sinc,
+        we encourage using using numpy.sinc instead which returns the
+        same result.
```

Comment:
I think this would be a more standard way to write this.

```suggestion
        `scipy.special.sinc` is a deprecated alias for `numpy.sinc` and will be
        removed in SciPy 1.14. To silence this warning, use `numpy.sinc` instead.
        Deprecated in SciPy 1.12.
```